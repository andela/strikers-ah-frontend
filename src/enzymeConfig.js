import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import 'jest-localstorage-mock';

export default Enzyme.configure({
  adapter: new Adapter(),
});
