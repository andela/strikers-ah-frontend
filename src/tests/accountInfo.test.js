import React from 'react';
import { shallow } from 'enzyme';
import '../enzymeConfig';
import AccountInfo from '../components/common/accountInfo';

describe('<AccountInfo />', () => {
  let props;
  beforeEach(() => {
    props = {
      articles: [{}],
      owner: false,
      showAddImageForm: { toggleForm: jest.fn(), visible: true },
      name: '',
    };
  });
  it('renders without crashing', () => {
    const component = shallow(<AccountInfo {...props} />);
    expect(component).toMatchSnapshot();
  });
  it('renders without crashing', () => {
    props.owner = true;
    const component = shallow(<AccountInfo {...props} />);
    expect(component).toMatchSnapshot();
  });
});
