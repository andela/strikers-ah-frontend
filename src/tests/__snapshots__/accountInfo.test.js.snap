// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AccountInfo /> renders without crashing 1`] = `ShallowWrapper {}`;

exports[`<AccountInfo /> renders without crashing 2`] = `ShallowWrapper {}`;
